components/builder-worker/src/error.rs

```diff
@@ -82,6 +86,12 @@ impl From<git2::Error> for Error {
     }
 }
 
+impl From<bldr_core::Error> for Error {
+    fn from(err: bldr_core::Error) -> Error {
+        Error::BuilderCore(err)
+    }
+}
+
```

Comment:
I've heard @reset say before that this is a pattern we should avoid going forward, and instead use things like `map_err` at the call site to convert errors from one type to another.

Reply:
Correct - this is not used, I just added it to stay consistent with the rest of the errors.